_episodes/01-intro.md

```diff
@@ -172,7 +172,7 @@ weight in kilograms is now: 65.0
 > ~~~
 > {: .output}
 >
-> ![Updating a Variable](../fig/python-sticky-note-variables-03.svg)
+> ![Updating the weight_kg variable with 100.0 as the value; weight_lb variable equals to 143.0](../fig/python-sticky-note-variables-03.svg)
```

Comment:
Since the text before and after the image indicates `weight_kg` has updated and `weight_lb` has not, what do you think about making this follow the pattern of intro image 01 and 02:
"Value of 100.0 with weight_kg name stuck on it, and value of 143.0 with weight_lb name stuck on it"?